go.sum

```diff
@@ -10,6 +10,10 @@ github.com/containernetworking/cni v0.8.0 h1:BT9lpgGoH4jw3lFC7Odz2prU5ruiYKcgAjM
 github.com/containernetworking/cni v0.8.0/go.mod h1:LGwApLUm2FpoOfxTDEeq8T9ipbpZ61X79hmU3w8FmsY=
 github.com/coreos/go-iptables v0.4.5 h1:DpHb9vJrZQEFMcVLFKAAGMUVX0XoRC0ptCthinRYm38=
 github.com/coreos/go-iptables v0.4.5/go.mod h1:/mVI274lEDI2ns62jHCDnCyBF9Iwsmekav8Dbxlm1MU=
+github.com/coreos/go-iptables v0.4.6-0.20200318170312-12696f5c9108 h1:H1IVw9/BmB880YZvqWEGuRVk/EUn4OukBjD7HW+zEYQ=
+github.com/coreos/go-iptables v0.4.6-0.20200318170312-12696f5c9108/go.mod h1:/mVI274lEDI2ns62jHCDnCyBF9Iwsmekav8Dbxlm1MU=
```

Comment:
Nit: we don't need these two lines.  `go mod tidy` will likely take them out.

Reply:
Pushed a new commit with tidy, let me know if you want a squash